doc/source/standard_selections.rst

```diff
@@ -0,0 +1,95 @@
+.. -*- coding: utf-8 -*-
+.. _standard-selections:
+
+==========================================
+Standard residues in MDAnalysis selections
+==========================================
+
+.. _protein-selection:
+
+Proteins
+========
+The residue names listed here are accessible via the "protein" keyword in the :ref:`selections`. 
+
+The below names are drawn from the CHARMM 27, OPLS-AA, GROMOS 53A6, AMBER 03, and AMBER 99sb*-ILDN force fields.
+
++------+------+------+------+------+------+------+------+
```

Comment:
Is there a way to generate this table programmatically from the MDAnalysis version that the UserGuide pertains to?

Although this is not likely to change soon, when it does I guarantee that someone will forget to update the UserGuide.

I think @jbarnoud has written code to, for instance, dynamically create the AUTHORS list in the docs.

Reply:
I think that sticking it in `conf.py` to run automatically would require installing MDAnalysis. If that isn't too heavy, this would be quite easy. Otherwise a script could be run manually from time-to-time to just regenerate `standard_selections.rst`?

Reply:
At least, you can have the table in a separate file that gets included here. This way, you can generate the table with a script without having to deal with the rest of the page in the script. I have an example in <https://github.com/MDAnalysis/mdanalysis/pull/2176>; it is a bit of a monster, but mostly because of what gets written in the external file.